msal4j-sdk/src/test/java/com/microsoft/aad/msal4j/ManagedIdentityTestConstants.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+class ManagedIdentityTestConstants {
```

Comment:
isn't there any utils where this can go?

Reply:
There's a `Constants` file in the main library and one in the integration tests folder, but there wasn't one for the unit tests. The unit tests have `TestHelper`, but that's more for helper methods than constants.

I thought about making a more generic constants file and that might happen in a later refactor, but since these are only used for MI tests I figured it'd be easier to keep it more focused for this PR.